src/local-settings.js

```diff
@@ -100,12 +100,17 @@ export function getLocalChainId() {
   return getLocalSetting(LOCAL_CHAIN_ID) || 1337
```

Comment:
I've taken the liberty of renaming some of the exposed methods to be more clear, without changing the underlying environment variables.

At a future point in time (e.g. if we evaluate snowpack / etc, we can make breaking changes :).